main-site/components/testimonials-section/testimonial-card/TestimonialCard.styles.ts

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+import { max } from '../../../../shared-ui/lib/responsive';
+import { colors } from '../../../../shared-ui/style/colors';
+import { H4, P } from '../../../../shared-ui/style/typography';
+
+const StyledTestimonialCardContainer = styled.div`
+  overflow: hidden;
+`;
+
+const StyledTestimonialCardBox = styled.div`
+  padding: 2.5em;
+  background-color: ${colors.TEXT_BOX};
+  margin: 2em;
+  border-radius: 2em;
+  width: 22em;
+  height: 28em;
+  @media ${max.tablet} {
+    width: 14em;
```

Comment:
Can we make this even wider since there's so much text we need to put on it? Like 18em or something and then if possible increase the font size at that screensize. 

You can add a different breakpoint for tabletSm or mobile to configure it more specifically for those screensizes